Zend/tests/002.phpt

```diff
@@ -40,62 +73,49 @@ call_user_func("test3", 1, 2);
 
 class test {
     static function test1($a) {
-        var_dump(func_get_arg(0));
-        var_dump(func_get_arg(1));
+        try {
+            var_dump(func_get_arg(0));
+        } catch (\Error $e) {
+            echo $e->getMessage() . \PHP_EOL;
+        }
+        try {
+            var_dump(func_get_arg(1));
+        } catch (\Error $e) {
+            echo $e->getMessage() . \PHP_EOL;
+        }
     }
 }
 
 test::test1(1);
-var_dump(func_get_arg(1));
+try {
+    var_dump(func_get_arg(1));
+} catch (\Error $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
 
 echo "Done\n";
 ?>
 --EXPECTF--
-Warning: func_get_arg():  The argument number should be >= 0 in %s on line %d
-bool(false)
-
-Warning: func_get_arg():  Argument 0 not passed to function in %s on line %d
-bool(false)
-
-Warning: func_get_arg():  Argument 1 not passed to function in %s on line %d
-bool(false)
-
-Warning: func_get_arg():  The argument number should be >= 0 in %s on line %d
-bool(false)
+func_get_arg(): Argument #1 ($arg_num) must be greater than or equal to 0
+func_get_arg(): Argument 0 not passed to function
```

Comment:
Is this zero-indexing how it seems? 🤔

Reply:
Yes, because the argument list starts from 0 for ``func_get_arg()`` kinda odd but it's documented.